app/serializers/user_serializer.rb

```diff
@@ -160,6 +161,14 @@ def second_factor_backup_enabled
     object.backup_codes_enabled?
   end
 
+  def include_second_factor_remaining_codes?
+    (object&.id == scope.user&.id) && object.backup_codes_enabled?
+  end
+
+  def second_factor_remaining_codes
```

Comment:
Wondering if this shouldn’t be `second_factor_remaining_backup_codes` what do you think?

Reply:
Agree, for the sake of consistency. Will add a change.